### DISCUSSION TITLE
IOTA Firefly, Manage Stake function not working

### DISCUSSION THREAD

**Discussion by LorenzoCrypto1** [iotaledger/firefly#2137] [Help & Support] [answered]
I have MIOTA in my unstaked funds that I am trying to migrate to the staked wallet. However, when I click on 'Manage Stake' in Firefly (under the Staking tab), the Syncing and Merge options are greyed out. I am unable to transfer this balance to that being actively staked. 

Currently using Firefly v 1.3.2 via a Ledger Nano X. 

Has anyone else been experiencing this issue?

**Comment by tridavid82** [comment]
Hola, yo tengo el mismo problema. 
Mande a mi cartera nuevos miotas en la cual ya tenia el staking activado y ahora no me deja ponerlos también en staking.
Las opciones de sincronización y combinación están atenuadas y no me deja pincharlas.
Tampoco tengo ninguna opción mas para salir del staking y después entrar con todo.
Alguien conoce alguna solución?

**Comment by MiKeF05** [comment, marked as answer]
Hello, you can try to create a new wallet inside the profile, transfer all funds to this wallet and stake again with the new wallet.

**Reply by tridavid82** [reply]
Muchas gracias por la solución.
Problema resuelto.
